scripts/create-project.sh

```diff
@@ -27,6 +27,9 @@ cat - >> .defaults.env <<END
   CONTENTPOOL_BASE_URL=http://example.contentpool-project.localdev.space
 END
 
+# Update file entity to dev version for 8.7 compatibility
+composer require drupal/file_entity:2.x-dev
```

Comment:
Why not set that to composer.json?

Reply:
because the satellite does not directly require file_entity and it does not hard-code / enforce 8.7 at all. also requiring dev-versions always sucks. better to stay with releases, so once the next rlease is here we can just drop that.